src/plugins/condition/ConditionManager.js

```diff
@@ -48,6 +52,9 @@ export default class ConditionManager extends EventEmitter {
     update(newDomainObject) {
```

Comment:
If we have one conditionManager per conditionSet, why do we need to update this condition manager? I am only concerned with that part that calls this.destroy on update and then resubscribes to changes on the same conditionSet domain Object.